platform/platform-filter-delegate/pom.xml

```diff
@@ -91,23 +118,23 @@
                             <rules>
                                 <rule>
                                     <element>BUNDLE</element>
+                                    <!--These ratios are low because the FilterInjector class has low unit test coverage.-->
                                     <limits>
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.32</minimum>
+                                            <minimum>0.99</minimum>
```

Comment:
Lets bump these down to 97 or so, to allow for some discrepancies between operating systems.